JsonSchema.Generation/Generators/ObjectSchemaGenerator.cs

```diff
@@ -43,7 +43,7 @@ public void AddConstraints(SchemaGenerationContextBase context)
 			var memberAttributes = member.GetCustomAttributes().ToList();
 #pragma warning disable 8600 // Assigning null to non-null
 			// ReSharper disable once AssignNullToNotNullAttribute
-			var ignoreAttribute = (Attribute)memberAttributes.OfType<JsonIgnoreAttribute>().FirstOrDefault() ??
+			var ignoreAttribute = (Attribute)memberAttributes.OfType<JsonIgnoreAttribute>().Where(a=>a.Condition == JsonIgnoreCondition.Always).FirstOrDefault() ??
```

Comment:
I fear this will hide the `[JsonIgnore]` attribute if the ignore condition isn't `Always`.  Would it be better to get the attribute if it exists at all, _then_ check the condition?

Reply:
Right, but I'm concerned doing this in a `.Where()` will _completely_ skip that the attribute is there, which is _not_ what we want if one of the other values is used.  I think it's better to get the attribute, then have the check for its value in a separate `if` statement.

Reply:
I’m really sorry, but I don’t understand what you are requesting in terms of behavior change.

The old code check for the existence of `System.Text.Json.[JsonIgnore]` or your custom `[JsonExclude]`, (in a very compact way) and ignores the whole property with `continue;` on the next line. The individual attributes are never read.

Reply:
Aha, so you were talking about the what would happen if both `JsonIgnore` and `JsonExclude`, would be set? Sorry for not understanding.

Should probably include some tests for all the possible combinations of the attributes, but wanted to clarify intended behavior first.